lib/Importer/ONNXIFILoader.cpp

```diff
@@ -135,36 +135,71 @@ std::unique_ptr<ModelLoader> ModelLoader::parse(
   return loader;
 }
 
-std::unique_ptr<std::pair<Kinded::Kind, ElemKind>>
+std::vector<std::pair<Kinded::Kind, ElemKind>>
 ModelLoader::parseOperator(const void *onnxModel, size_t onnxModelSize) {
-
+  std::vector<std::pair<Kinded::Kind, ElemKind>> result;
   ONNX_NAMESPACE::ModelProto modelDef;
-  if (ONNXModelLoader::loadProto(modelDef, onnxModel, onnxModelSize)) {
-    return nullptr;
+  if (!ONNXModelLoader::loadProto(modelDef, onnxModel, onnxModelSize)) {
+    return result;
   }
 
   ONNX_NAMESPACE::GraphProto graph = modelDef.graph();
 
   // Only single operator is allowed to be in the onnxModel.
   if (graph.node_size() != 1) {
-    return nullptr;
+    return result;
   }
 
-  std::unique_ptr<std::pair<Kinded::Kind, ElemKind>> result;
   const std::string &operation = graph.node(0).op_type();
 
   // Quantized and non-quantized operations are handled by
   // different ONNX operators, for now only handle fp32.
   // TODO: Add more operators.
-  if (operation == "conv") {
-    result.reset(new std::pair<Kinded::Kind, ElemKind>(
+  if (operation == "BatchNormalization") {
+    result.push_back(std::pair<Kinded::Kind, ElemKind>(
```

Comment:
nit: you can just do `result.emplace_back(Kinded::Kind::BatchNormalizationNodeKind, ElemKind::FloatTy)`